src/features/comments/components/Comment.tsx

```diff
@@ -0,0 +1,178 @@
+import {
+  Box,
+  Card,
+  CardContent,
+  Typography,
+  IconButton,
+  Button,
+  Avatar,
+} from "@mui/material";
+import { AddComment, Favorite } from "@mui/icons-material";
+import { BaseCommentPropType } from "../models/Comments.model";
```

Comment:
is this being used?

Reply:
No but we will when we add comments comments endpoint because we will need to distinguish the differences between the post and comment comments but I guess we can temporarily remove it, I was just waiting for someone to build the comments comments endpoint.